src/spikeinterface/postprocessing/localization_tools.py

```diff
@@ -667,14 +667,14 @@ def compute_location_max_channel(
     """
     Localize a unit using max channel.
 
-    This use inetrnally get_template_extremum_channel()
+    This uses interrnally `get_template_extremum_channel()`
```

Comment:
Nice to correct a typo with another smaller typo! You almost got us all fooled :P
```suggestion
    This uses internally `get_template_extremum_channel()`
```

Reply:
That was my fault. Sometimes I type too fast too! my bad :)